web/src/main/java/com/epam/rft/atsy/web/controllers/rest/StateHistoryController.java

```diff
@@ -107,8 +107,14 @@ public ResponseEntity saveOrUpdate(@RequestParam Long applicationId,
       }
 
       if (stateId != null && stateId == 1) {
```

Comment:
Write comment about this line, because it seems some kind of magic.

The reasons are described in a previous comment.